SQLAlchemy_db_connector/repositories/orders_repository.py

```diff
@@ -0,0 +1,38 @@
+from SQLAlchemy_db_connector import connection_db
+from SQLAlchemy_db_connector.models.orders import Orders
+from SQLAlchemy_db_connector.models.products import Products
+from sqlalchemy import select, delete, update
+
+
+class OrdersRepository:
```

Comment:
does it work in separate class with pony?

Reply:
what is a pony? I built the framework according to the lesson and spent a lot of time studying the documentation

Reply:
I understand, pony in the next lesson. I will watch and implement in this framework